### PR TITLE
Fix kernel headers

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

File: layers/meta-resin-jetson/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bbappend [hunk 1]
```diff
@@ -0,0 +1 @@
+SRCREV = "v0.0.10"
```
Comment (agherzan) [inline layers/meta-resin-jetson/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bbappend]: Why is this change here and not in meta resin?

Comment (agherzan) [reply layers/meta-resin-jetson/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bbappend]: I found the PR for meta-resin too. I don't like that this is overwriting the value from meta-resin which would be bad in case of meta-resin updated to a version later than 0.0.10.

**Comment by telphan** [comment]
@agherzan then release meta-resin now and I will just jump to the revision.

**Comment by agherzan** [comment]
@telphan v2.8.1

**Comment by telphan** [comment]
@agherzan Changed PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Change log
 -----------
 
+# v2.8.1+rev1
+## 2017-12-1
```
Comment (agherzan) [inline CHANGELOG.md]: Use "2017-12-01"

**Comment by imrehg** [comment]
Can't really comment on this, so no review from my part.
